guides/release/typescript/core-concepts/invokables.md

```diff
@@ -701,17 +696,134 @@ function isOrdered(type: 'ordered' | 'unordered'): type is 'ordered' {
 If you are using Glint, when this component is invoked, the `@type` argument will determine what kinds of modifiers are legal to apply to it. For example, if you defined a modifier `reverse` which required an `HTMLOListElement`, this invocation would be rejected:
 
 ```handlebars
-<List @items={{array 1 2 3}} @type='unordered' {{reverse}} as |item|>
-  The item is
-  {{item}}.
-</List>
+<List @items={{array 1 2 3}} @type='unordered' {{reverse}} />
```

Comment:
This wouldn't type-check since we've passed an array of numbers when an array of strings is expected.

```suggestion
<List @items={{array "kdagnan" "gitKrystan" "wagenet"}} @type='unordered' {{reverse}} />
```